### PR TITLE
merge coreth 0.8.9-rc.0

### PR DESCRIPTION
**Pull request by ceyonur**

### CODE REVIEW

File: core/rawdb/schema.go [hunk 1]
```diff
@@ -61,6 +61,13 @@ var (
 	// offlinePruningKey tracks runs of offline pruning
 	offlinePruningKey = []byte("OfflinePruning")
 
+	// populateMissingTriesKey tracks runs of trie backfills
+	populateMissingTriesKey = []byte("PopulateMissingTries")
+
+	// pruningDisabledKey tracks whether the node has ever run in archival mode
+	// to ensure that a user does not accidentally corrupt an archival node.
+	pruningDisabledKey = []byte("PruningDisabled")
```
Comment (patrick-ogrady) [inline core/rawdb/schema.go:69]: I forgot this was not carried over yet.

File: plugin/evm/vm.go [hunk 2]
```diff
@@ -250,6 +253,9 @@ func (vm *VM) Initialize(
 	ethConfig.AllowUnprotectedTxs = vm.config.AllowUnprotectedTxs
 	ethConfig.Preimages = vm.config.Preimages
 	ethConfig.Pruning = vm.config.Pruning
+	ethConfig.PopulateMissingTries = vm.config.PopulateMissingTries
+	ethConfig.PopulateMissingTriesParallelism = vm.config.PopulateMissingTriesParallelism
+	ethConfig.AllowMissingTries = vm.config.AllowMissingTries
 	ethConfig.SnapshotAsync = vm.config.SnapshotAsync
 	ethConfig.SnapshotVerify = vm.config.SnapshotVerify
 	ethConfig.OfflinePruning = vm.config.OfflinePruning
```
Comment (darioush) [inline plugin/evm/vm.go:261]: There's a comment missing from line 265:
`	// Create directory for offline pruning`

File: README.md [hunk 3]
```diff
@@ -554,25 +574,27 @@ _The `vm-id` was generated by calling `subnet-cli create VMID subnetevm`. You ca
...
-  "network-id":"fuji",
-  "health-check-frequency":"2s",
-  "log-display-level":"INFO",
-  "log-level":"INFO",
-  "whitelisted-subnets":"<PROVIDED BY SPACES-CLI>"
+  "network-id": "fuji",
+  "health-check-frequency": "2s",
+  "log-display-level": "INFO",
+  "log-level": "INFO",
+  "whitelisted-subnets": "<PROVIDED BY SPACES-CLI>"
```
Comment (darioush) [inline README.md:592]: should this be SPACES-CLI?

File: README.md [hunk 4]
```diff
@@ -775,4 +1036,4 @@ contract WWAGMI{
 }
 ```
 
-[become a Fuji Validator]: https://docs.avax.network/build/tutorials/nodes-and-staking/staking-avax-by-validating-or-delegating-with-the-avalanche-wallet
+[become a fuji validator]: https://docs.avax.network/build/tutorials/nodes-and-staking/staking-avax-by-validating-or-delegating-with-the-avalanche-wallet
```
Comment (darioush) [inline README.md:1039]: should we keep `Fuji`?

**Review by darioush — APPROVED** [review]
LGTM after pull from master

File: README.md (hunk 3 above)
Comment (ceyonur) [reply README.md:592]: I think this section is under "### Run `subnet-cli wizard`" so it still shows how to use `subnet-cli`

File: README.md (hunk 4 above)
Comment (ceyonur) [reply README.md:1039]: nope, this looks weird :D

File: core/blockchain.go [hunk 5]
```diff
@@ -525,20 +538,18 @@ func (bc *BlockChain) Stop() {
 		return
 	}
 
-	log.Info("Shutting down state manager")
+	log.Info("Shutting down state maanger")
```
Comment (patrick-ogrady) [inline core/blockchain.go:541]: typo

**Review by patrick-ogrady — APPROVED** [review]

**Pull request merged**